Library/Homebrew/extend/ARGV.rb

```diff
@@ -88,8 +88,9 @@ def next
   end
 
   def value(arg)
-    arg = find { |o| o =~ /--#{arg}=(.+)/ }
-    $1 if arg
+    return if arg.empty? || arg.index("-") || arg.index("=")
+    match = find { |opt| opt.start_with?("--#{arg}=") }
```

Comment:
This value is what's going to become the output of the method, after some more intermediate processing. I use `out` for variables like that. Does that fit in with Homebrew style? Maybe `val` (as an abbreviation for `value`, since it's what the `value` method returns) or `optarg` (short for "option argument", and the conventional name for this thing in `getopt()`).